CONTRIBUTING.md

```diff
@@ -47,6 +47,12 @@ contributors.
 - Add a `replace` directive at the root `go.mod` file so your component is
   included in the build of the contrib executable.
 - Add your component to `versions.yaml`.
+- All components must be included in [`internal/components/`](./internal/components) 
+  and in the respective testing harnesses. To align with the test goal of the project. 
```

Comment:
> To align with the test goal of the project.

This still looks out of place. Perhaps:

> To align with the test goal of the project,
> components must be testable within the framework defined within the folder.

Reply:
Yeah, that reads better to me :) 

Sorry, my dyslexia stopped me from seeing the difference in punctuation.

Reply:
No need to be sorry at all, you will probably see me making similar mistakes elsewhere :-)